inotify.go

```diff
@@ -245,6 +245,15 @@ func (w *Watcher) readEvents() {
 
 			mask := uint32(raw.Mask)
 			nameLen := uint32(raw.Len)
+
+			if mask&unix.IN_Q_OVERFLOW != 0 {
+				select {
+				case w.Errors <- errors.New("inotify queue overflow"):
```

Comment:
Just FYI, the Windows code already has its own special way of handling overflows (see sysFSQOVERFLOW).  It might be worth unifying handling of overflow on Linux and Windows.  That said, I'm a bit swamped in the next week or two, so I'm unlikely to revise this patch until next month at the earliest.  It might make sense to merge this as-is (since it seems no worse than the current behavior), and perhaps unify overflow handling as a separate patch later.